### PR TITLE
Adding double quotes to Ruby script

### PR DESCRIPTION
**Pull request by datanerd**
### Why:
Not having the path and App ID enclosed in double quotes will result in errors.

Closes [issue link]

### What's being changed:
Added double quotes around the path to the certificate file and AppID

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by affrae — CHANGES_REQUESTED** [review]
Thank you @andrekolodochka  - can we also clarify in the actual documentation underneath where it mentions `iss` that `YOUR_APP_ID` needs to be enclosed in double quotes as a string value?

**Review by affrae — DISMISSED** [review]
Notwithstanding any concerns from @github/docs I am fine with this! Thank you @andrekolodochka!

**Review by pablo6116 — COMMENTED** [review]
andrekolodochka-patch-2

**Comment by Pangkok** [comment]
pablo6116 ***@***.***> 于 2021年9月10日周五 09:52写道：

**Comment by pablo6116** [comment]
andrekolodochka-patch-2

**Comment by Pangkok** [comment]
pablo6116 ***@***.***> 于 2021年9月10日周五 10:09写道：

**Comment by Pangkok** [comment]
https://www.reddit.com/r/Coinex

**Comment by affrae** [comment]
ummmmm

**Comment by ramyaparimi** [comment]
@andrekolodochka 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/developers/apps/building-github-apps/authenticating-with-github-apps.md [hunk 1]
```diff
@@ -82,14 +82,14 @@ payload = {
...
-  iss: YOUR_APP_ID
+  iss: "YOUR_APP_ID"
 }
 
 jwt = JWT.encode(payload, private_key, "RS256")
 puts jwt
 ```
 
-`YOUR_PATH_TO_PEM` and `YOUR_APP_ID` are the values you must replace.
+`YOUR_PATH_TO_PEM` and `YOUR_APP_ID` are the values you must replace and make sure to enclose them in double quotes.
```
Comment (lecoursen) [inline content/developers/apps/building-github-apps/authenticating-with-github-apps.md:92]:
```suggestion
`YOUR_PATH_TO_PEM` and `YOUR_APP_ID` are the values you must replace. Make sure to enclose the values in double quotes.
```

**Review by lecoursen — COMMENTED** [review]
Thanks so much @andrekolodochka! I'm going to apply one tiny suggestion and then merge. ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**